ratingService/src/main/resources/application.properties

```
@@ -0,0 +1,8 @@
+spring.application.name=ratingService
+spring.data.mongodb.uri=mongodb://localhost:27017/rating_service_db
```

Стоит передавать значение в переменной среды, т.к. url может меняться в зависимости от окружения.
```suggestion
spring.data.mongodb.uri=${MONGODB_URL:mongodb://localhost:27017/rating_service_db}
```